config/routes.js

```diff
@@ -52,6 +52,7 @@ module.exports.routes = {
   '/user/logout': 'userController.logout',
   '/user/status': 'userController.status',
 
+  'GET /api/status': 'StatusController.getStatus',
```

Comment:
I love that you jumped right in to sails and just started hooking things up-- really nice work!